nucypher/config/migrations/__init__.py

```diff
@@ -0,0 +1,18 @@
+from collections import OrderedDict
+
+from .configuration_v1_to_v2 import configuration_v1_to_v2
+from .configuration_v3_to_v4 import configuration_v3_to_v4
+from .configuration_v4_to_v5 import configuration_v4_to_v5
+from .configuration_v4_to_v6 import configuration_v4_to_v6
+from .configuration_v5_to_v6 import configuration_v5_to_v6
+
+MIGRATIONS = OrderedDict(
+    {
+        (1, 2): configuration_v1_to_v2,
+        (2, 3): None,  # (no-op)
+        (3, 4): configuration_v3_to_v4,
+        (4, 5): configuration_v4_to_v5,
+        (4, 6): configuration_v4_to_v6,
```

Comment:
On second thought this can be excluded since we're already processing these migrations in series, this one will never be applied from the CLI command.

Reply:
really, line 15 should come before 14

Reply:
I've also realized that anyone who has configuration file v1will be stuck without the 2 -> 3 migration.  Fortunately I do not think this is really an issue since most node operators are likely running configuration v4.

Reply:
yeah, this is still an improvement. Those people were already in a painful place

Reply:
Agreed